gradle/libs.versions.toml

```diff
@@ -23,6 +23,8 @@ auto-value-annotations = "com.google.auto.value:auto-value-annotations:1.10.4"
 checkstyle = "com.puppycrawl.tools:checkstyle:10.12.5"
 commons-math3 = "org.apache.commons:commons-math3:3.6.1"
 conscrypt = "org.conscrypt:conscrypt-openjdk-uber:2.5.2"
+# Update notes / 2023-07-19 sergiitk:
+#     Upgrade is blocked by https://github.com/grpc/grpc-java/issues/10396.
```

Comment:
Can you specify whether it is `cronet-api` or `conscrypt` that is blocked?

Reply:
Thanks, done!